horovod/_keras/__init__.py

```diff
@@ -30,7 +31,8 @@ def create_distributed_optimizer(keras, optimizer, name, device_dense, device_sp
                                  compression, sparse_as_dense, gradient_predivide_factor,
                                  op, backward_passes_per_step=1,
                                  average_aggregated_gradients=False,
-                                 groups=None, process_set=hvd.global_process_set):
+                                 groups=None, process_set=hvd.global_process_set,
+                                 scale_local_gradients=True):
```

Comment:
Instead of a bool, can we use floating points?

Reply:
can you elaborate? the floating point will be the scaling value?

Reply:
yes